parsers/sql.c

```diff
@@ -294,6 +300,7 @@ static const keywordTable SqlKeywordTable [] = {
 	{ "drop",							KEYWORD_drop			      },
 	{ "else",							KEYWORD_else			      },
 	{ "elseif",							KEYWORD_elseif			      },
+	{ "elsif",							KEYWORD_elsif			      },
```

Comment:
Could you add an entry for `KEYWORD_elsif` to 
```
static struct SqlReservedWord SqlReservedWord [SQLKEYWORD_COUNT] = {
	/*
	 * RESERVED_BIT: MYSQL & POSTGRESQL&SQL2016&SQL2011&SQL92 & ORACLE11g&PLSQL & SQLANYWERE
	 *
	 * {  0  } means we have not inspect whether the keyword is reserved or not.
	 */
```

Reply:
Done, thanks for noticing. I'll then squash the commits when you think there's not more work to do.

Reply:
Yes, please. Could you remove [c3b0324](https://github.com/universal-ctags/ctags/pull/3654/commits/c3b0324af1b9db57dc2c5a913bb8d7a650328654) and [9f532d3](https://github.com/universal-ctags/ctags/pull/3654/commits/9f532d3f72d358be7ecf0d932f098f5f72b13c10)?

Reply:
Done.